### DISCUSSION TITLE
Question on supplying DiaNN with a spectral library

### DISCUSSION THREAD

**Discussion by lubwen** [vdemichev/DiaNN#341] [Q&A] [answered]
Hi,

I would like to use DiaNN for my DIA data analysis. Because my data contain a protein modification that is not supported by DiaNN's spectral prediction algorithm, I would like to supply DiaNN with my own spectral library. My question is: what modification format is accepted by DiaNN for the 'PeptideSequence' column in the supplied TSV file?

For example, for peptide QLGCGQALAAPGEAHFGPGR with a modification of +324 mass on cysteine (C), is the following peptide accepted by DiaNN?  
QLGC[+324]GQALAAPGEAHFGPGR

Thanks so much for provide a nice tool for DIA data analysis.

**Comment by vdemichev** [comment, marked as answer]
Yes, anything in parentheses works as a modification name.

Vadim
